scripts/parse_generic_metadata.py

```diff
@@ -1,6 +1,6 @@
 #!/usr/bin/env python3
 # pylint: disable=too-many-instance-attributes,too-many-locals,unused-argument,no-self-use,wrong-import-order,unused-argument,too-many-arguments
-from typing import List, Optional, Tuple
+from typing import Any, Dict, List, Optional, Tuple
```

Comment:
This file is completely contained within the parser now right?
A quick way to remove the code but keep the functionality could be to call the main function within this file, eg:

```python
if __name__ == '__main__':
    from sample_metadata.parser.generic_metadata_parser import main
    main() # pylint: disable=no-value-for-parameter
```

Reply:
Yeah I didn't want to delete it before full copying over then I forgot lol.